frontend/packages/operator-lifecycle-manager/src/components/operator-group.tsx

```diff
@@ -12,56 +12,61 @@ import { AsyncComponent } from '@console/internal/components/utils/async';
 import { getActiveNamespace } from '@console/internal/actions/ui';
 import { OperatorGroupModel } from '../models';
 import { OperatorGroupKind, SubscriptionKind, InstallModeType } from '../types';
+import { useTranslation } from 'react-i18next';
 
 export const targetNamespacesFor = (obj: K8sResourceKind) =>
-  _.get(obj, ['metadata', 'annotations', 'olm.targetNamespaces']);
+  obj?.metadata?.annotations?.['olm.targetNamespaces'];
 export const operatorNamespaceFor = (obj: K8sResourceKind) =>
-  _.get(obj, ['metadata', 'annotations', 'olm.operatorNamespace']);
+  obj?.metadata?.annotations?.['olm.operatorNamespace'];
 export const operatorGroupFor = (obj: K8sResourceKind) =>
-  _.get(obj, ['metadata', 'annotations', 'olm.operatorGroup']);
-
-export const NoOperatorGroupMsg: React.SFC = () => (
-  <MsgBox
-    title="Namespace Not Enabled"
-    detail={
-      <p>
-        The Operator Lifecycle Manager will not watch this namespace because it is not configured
-        with an OperatorGroup.{' '}
-        <Link to={`/ns/${getActiveNamespace()}/${referenceForModel(OperatorGroupModel)}/~new`}>
-          Create one here.
-        </Link>
-      </p>
-    }
-  />
-);
-
-type RequireOperatorGroupProps = {
-  operatorGroup: { loaded: boolean; data?: OperatorGroupKind[] };
+  obj?.metadata?.annotations?.['olm.operatorGroup'];
+
+export const NoOperatorGroupMsg: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <MsgBox
+      title="Namespace Not Enabled"
+      detail={
+        <p>
+          {t(
+            'olm~The Operator Lifecycle Manager will not watch this namespace because it is not configured with an OperatorGroup.',
```

Comment:
```suggestion
            'olm~The Operator Lifecycle Manager will not watch this Namespace because it is not configured with an OperatorGroup.',
```